display_books_promise.js

```diff
@@ -0,0 +1,65 @@
+let BookData = new Array();
+fetch("random_books.json")
+  .then((response) => response.json())
+  .then((data) => {
+    BookData = data;
+    insertDataInHtml(BookData);
+  })
+  .catch((error) => console.log(error));
+
+document.getElementById("btn-search-id").onclick = IdButton;
```

Comment:
Read about how onClick event is added in ES6.
The above line can be written as - 
```
document.getElementById('btn-search-id').addEventListener('click', IdButton);
```
The function naming is not self explanatory. What is the work of IdButton?

Reply:
Similarly there are other instances where improvement is needed.

Reply:
Resolved